Documentation/ceph-cluster-crd.md

```diff
@@ -135,7 +135,7 @@ The following storage selection settings are specific to Ceph and do not apply t
 - `walSizeMB`:  The size in MB of a bluestore write ahead log (WAL). Include quotes around the size.
 - `journalSizeMB`:  The size in MB of a filestore journal. Include quotes around the size.
 - `osdsPerDevice`**: The number of OSDs to create on each device. High performance devices such as NVMe can handle running multiple OSDs. If desired, this can be overridden for each node and each device.
-- `encryptedDevice`**: Encrypt OSD volumes using dmcrypt ("true" or "false"). By default this option is disabled.
+- `encryptedDevice`**: Encrypt OSD volumes using dmcrypt ("true" or "false"). By default this option is disabled. See http://docs.ceph.com/docs/master/ceph-volume/lvm/encryption/ for more information on encryption in Ceph.
```

Comment:
shall we link to nautilus instead of master?

Reply:
Side note: Are we going to remember to update links in these when we support newer Ceph versions? I wonder if we should make a release document with a checklist of all the things that should be looked for when updating docs for a new release...

Reply:
Upstream Ceph documentation is tentatively going to move to a new URL scheme where we have an unversioned docs URL that redirects to the latest release, and each page allows easy switching between versions.